docusaurus/docs/user-docs/content-manager/saving-and-publishing-content.md

```diff
@@ -42,6 +42,22 @@ Published contents can be unpublished, switching back to being drafts again.
 
 To unpublish content, click on the **Unpublish** button in the top right corner of the content editor.
 
+### Bulk publish and unpublish <BetaBadge />
+
+Selecting multiple entries from the Content Manager's list view will display additional buttons to publish or unpublish several entries simultaneously.
+
+![Selecting entries for bulk publish/unpublish](/img/assets/content-manager/bulk-publish.png)
+
+To publish or unpublish several entries:
+
+1. From the list view of the Content Manager, select your entries to publish or unpublish by ticking the box on the left side of the entries' record.
+2. Depending on what you want to achieve, click one of the **Publish** or **Unpublish** buttons located above the header of the table. When selecting only published entries, only the **Unpublish** button is shown, while when selecting only draft entries, only the **Publish** button is shown.
+3. In the confirmation dialog box, confirm your choice by clicking on the **Publish** or **Unpublish** button.
```

Comment:
I see below that for the bulk delete action, you have this warning:

>✋ Caution
>If the [Internationalization plugin](https://documentation-git-user-bulk-publish-strapijs.vercel.app/user-docs/plugins/strapi-plugins#internationalization-plugin) is installed, entries can only be deleted one locale at the time.

The same thing is true for bulk publish and unpublish: if localization is enabled for the content type, then the actions you make will only apply to the currently selected locale